src/game_destiny.cpp

```diff
@@ -0,0 +1,342 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Headers
+#include "game_destiny.h"
+
+#ifndef EMSCRIPTEN
+#include "exe_reader.h"
+#endif // !EMSCRIPTEN
+#include "output.h"
+
+using Destiny::InterpretFlag;
+using Destiny::MainFunctions::Interpreter;
+
+using lcf::ToString;
+using lcf::rpg::EventCommand;
+using lcf::rpg::SaveEventExecFrame;
+
+
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+//				Game_Destiny implementations
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+void Game_Destiny::Load()
+{
+	// Do not load Destiny whether player cannot find "Destiny.dll"
+	if (! FileFinder::Game().Exists(DESTINY_DLL))
```

Comment:
Can you add ``#include "filefinder.h"`` to the headers?  This fails to build on emscripten. Guess ``exe_reader`` indirectly pulls that header in and that include is skipped for the emscripten build.

Reply:
You mean this:
```cpp
// game_destiny.cpp

// Headers
#include "game_destiny.h"
#include "filefinder.h"
```

or this?

```cpp
// game_destiny.h

#ifndef EP_GAME_DESTINY_H
#define EP_GAME_DESTINY_H

#include <cstdint>
#include <sstream>
#include <vector>

#include "filefinder.h"
#include "lcf/rpg/saveeventexecframe.h"
```

Reply:
add it to the .cpp file (so first option)

Reply:
Done! =D